src/loaders/loadModelsAsync.ts

```diff
@@ -110,7 +130,7 @@ export async function loadDaeAsync({
 async function loadFileContentsAsync(loader, uri, funcName): Promise<any> {
   try {
     const fileContents = await readAsStringAsync(uri);
-    return loader.parse(fileContents);
+    return new Promise((res, rej) => loader.parse(fileContents, uri, res, rej));
```

Comment:
GLTFLoader from three.js does not return anything, it hust calls the res & rej functions when done,
This might affect others loaders included, but i did not test them